ui/pages/Onboarding/OnboardingAddWallet.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactElement } from "react"
+import BackButton from "../../components/Shared/SharedBackButton"
+import SharedButton from "../../components/Shared/SharedButton"
+
+export default function OnboardingAddWallet(): ReactElement {
+  return (
+    <section>
+      <div className="wordmark" />
+      <BackButton />
+      <div className="primary_wrap">
+        <div className="choice_wrap">
+          <h1 className="serif_header">Explore Tally</h1>
+          <div className="subtitle">
+            Use one of your existing addresses to explore Tally. You will not be
+            able to sign transaction untill you upgrade it.
+          </div>
+          <SharedButton
+            type="primary"
+            size="medium"
+            linkTo="/onboarding/viewOnlyWallet"
+          >
+            Add address
+          </SharedButton>
+        </div>
+        <div className="or_divider">or</div>
+        <div className="choice_wrap">
+          <h1 className="serif_header">Import seed</h1>
+          <div className="subtitle">
+            Use an existing Metamask seed or any web3 wallet seed.
```

Comment:
```suggestion
            Use an existing Metamask seed; other web3 wallets coming soon!
```

Something like this :grimacing: